boards/arm/adafruit_grand_central_m4_express/doc/index.rst

```diff
@@ -0,0 +1,191 @@
+.. _adafruit_grand_central_m4_express:
+
+Adafruit Grand Central M4 Express
+#################################
+
+Overview
+********
+
+The Adafruit Grand Central M4 Express is an ARM development board with the
+form factor of an Arduino Mega.
+It features 70 GPIO pins, a microSDHC slot and 8MiB of QSPI Flash.
+
+.. image:: img/adafruit_grand_central_m4_express.jpg
+     :align: center
+     :alt: Adafruit Grand Central M4 Express
+
+Hardware
+********
+
+- ATSAMD51P20A ARM Cortex-M4F processor at 120 MHz
+- 1024 KiB of flash memory and 256 KiB of RAM
+- 8 MiB of QSPI flash
+- A red user LED
+- A RGB "NeoPixel" / WS2812B LED
```

Comment:
The thing is that the Neopixel is connected to SERCOM 0 and 2 which are used by the SD card and the Console/UART.
Afaik it wouldn't work with SPI anyway since it is connected to SERCOM pad 2 for both which can't be used as MOSI.

The "worldsemi,ws2812-gpio" driver is apparently hardcoded for nordic chips so it is not an alternative.

Perhaps it could work with "zephyr,spi-bitbang" and "worldsemi,ws2812-spi" drivers but that is a bit of a mess.

Personally I don't need it so I just skipped it.
If it is important to you I can try it with the spi bitbangig driver though.

Reply:
> Personally I don't need it so I just skipped it. If it is important to you I can try it with the spi bitbangig driver though.

No, not that important :) I thought it would be a low hanging fruit, but looks like it's more tricky. I didn't realize the GPIO driver was nordic specific :/